opentelemetry-otlp/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Released 2025-Sep-25
 
 - Update `opentelemetry-proto` and `opentelemetry-http` dependency version to 0.31.0
 - Add HTTP compression support with `gzip-http` and `zstd-http` feature flags
+- Add retry with exponential backoff and throttling support for HTTP and gRPC exporters
```

Comment:
lets add some more details here so the user reading the changelog would know how to use this feature. (given this is experimental and opt-in).